packages/calypso-e2e/src/lib/test-account.ts

```diff
@@ -3,26 +3,29 @@ import path from 'path';
 import chalk from 'chalk';
 import { BrowserContext, Page } from 'playwright';
 import { TestAccountName } from '..';
-import { getAccountCredential, getAccountSiteURL, getCalypsoURL } from '../data-helper';
+import { getAccountSiteURL, getCalypsoURL } from '../data-helper';
+import { EmailClient } from '../email-client';
 import envVariables from '../env-variables';
+import { SecretsManager } from '../secrets';
 import { TOTPClient } from '../totp-client';
 import { LoginPage } from './pages/login-page';
-import type { AccountCredentials } from '../types/data-helper.types';
+// import type { AccountCredentials } from '../types/data-helper.types';
+import type { TestAccountCredentials } from '../secrets';
 
 /**
  * Represents the WPCOM test account.
  */
 export class TestAccount {
 	readonly accountName: TestAccountName;
-	readonly credentials: AccountCredentials;
+	readonly credentials: TestAccountCredentials;
```

Comment:
This is a broadening of the type definition for the variable `credentials`.

`AccountCredentials` narrowed the available fields to just `username, password, totpKey` and was defined separately in `DataHelper`. This was a carryover from the days where the SecretManager did not have type-checking.

Now that SecretsManager has type-checking, we are safe to use the entire returned value from SecretsManager when accessing a specific test account, ie.

```
const credentials = SecretManager.sercrets.testAccount.defaultUser`
```